app/org/maproulette/models/dal/TaskReviewDAL.scala

```diff
@@ -375,6 +375,65 @@ class TaskReviewDAL @Inject() (
     }
   }
 
+  /**
+    * Retrieve tasks geographically closest to the given task id wit the
+    * given set of search parameters.
+    */
+  def getNearbyReviewTasks(
```

Comment:
Any reason why this function is not built in the new framework?

Reply:
The method is still relying on TaskDal stuff and so would be complicated to convert over. Next step.

Reply:
Ok sounds good.